EDEngineer/Resources/Data/localization.json

```diff
@@ -3424,15 +3424,15 @@
     "Group ingredients?": {
       "es": null,
       "pt": null,
-      "de": null,
+      "de": "Gruppenzutaten?"

```

Comment:
requires a comma at the end of the line

Reply:
Ah jeez, first time editing something on Git. How do I change this now without pulling a new request?

Reply:
you can edit this file directly